avocado/utils/network/ports.py

```
@@ -137,31 +154,55 @@ def find_free_ports(
+
+        :param port: The port number to register.
+        :type port: int
+        :return: The registered port number if successful.
+        :rtype: int
+        :raises ValueError: If the port is already in use or cannot be
+                             registered.
+        """
         if (port not in self.retained_ports) and is_port_free(port, self.address):
             self.retained_ports.append(port)
         else:
             raise ValueError(f"Port {int(port)} in use")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)